client/my-sites/domains/map-domain/index.jsx

```diff
@@ -111,6 +113,12 @@ export class MapDomain extends Component {
 		}
 	}
 
+	checkIfDomainIsTransferrable( props ) {
```

Comment:
In the mapping step, you should be checking/streamlining the `mappable` case, not the transferrable step.

Reply:
Got it. I'll get after it!